app/views/shared/_manage_organization_header.html.erb

```diff
@@ -10,13 +10,12 @@
 <div class="d-flex flex-row justify-content-center">
     <div class="btn-group manage-organization-tabs" role="group" aria-label="Manage organization tabs">
         <%= link_to "Users", organization_users_path(@organization), class: "btn btn-outline-primary #{current_page_class(organization_users_path(@organization))}" %>
-
-        <% if can? :read, @organization.allowlisted_jwts.build %>
-          <%= link_to "Access tokens", organization_allowlisted_jwts_path(@organization), class: "btn btn-outline-primary #{current_page_class(organization_allowlisted_jwts_path(@organization))}" %>
+        <% if @organization.provider? && can?(:read, @organization.allowlisted_jwts.build) %>
```

Comment:
Consumer orgs need to be able to manage tokens so they can consume data via the API.
```suggestion
        <% if can? :read, @organization.allowlisted_jwts.build %>
```

Reply:
thanks; I was unsure about this — @ggeisler seemed to think otherwise in https://github.com/pod4lib/aggregator/issues/535.

Reply:
Well, I **assumed** consumers didn't need to do anything with tokens, obviously not understanding what tokens are for (though I guess I'm starting to understand..).